crates/kitsune_p2p/timestamp/Cargo.toml

```diff
@@ -14,7 +14,8 @@ edition = "2018"
 
 [dependencies]
 
-chrono = "0.4.6"
+chrono = { version = "0.4.6", features = ["serde"] }
```

Comment:
Not so relevant to this PR but just an FYI chrono has been deprecated and is considered security vulnerable. So we will have to move to the time crate soon.